kroxylicious-operator/install/01.ClusterRole.kroxylicious-operator-watched.yaml

```
@@ -33,6 +33,7 @@ rules:
     resources:
       - kafkaproxies/status
       - kafkaproxyingresses/status
+      - kafkaprotocolfilter/status
```

```suggestion
      - kafkaprotocolfilters/status
```
not sure if it's required, but yeh